app/components/channel_list/header/header.tsx

```diff
@@ -89,13 +125,7 @@ const ChannelListHeader = ({displayName, iconPad}: Props) => {
                     <CompassIcon
                         style={styles.plusIcon}
                         name={'plus'}
-                        onPress={async () => {
-                            const title = intl.formatMessage({id: 'browse_channels.title', defaultMessage: 'More Channels'});
-                            const closeButton = await CompassIcon.getImageSource('close', 24, theme.sidebarHeaderTextColor);
-                            showModal(Screens.BROWSE_CHANNELS, title, {
-                                closeButton,
-                            });
-                        }}
+                        onPress={onPress}
```

Comment:
onPress should be a prop of the parent `TouchableWithFeedback`

Reply:
Done